<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCDBSessionManager</class>
 <widget class="QDialog" name="GCDBSessionManager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>73</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select a profile for this session...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.png</normaloff>:/resources/goblinicon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="whatsThis">
        <string>All known XML profiles are shown here.</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showHelpButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/gcresources.qrc">
         <normaloff>:/resources/helpicon.png</normaloff>:/resources/helpicon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="whatsThis">
        <string>Select the profile displayed in the combo drop down.</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addNewButton">
       <property name="whatsThis">
        <string>Add a new XML profile.</string>
       </property>
       <property name="text">
        <string>Add New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addExistingButton">
       <property name="whatsThis">
        <string>Add an existing XML profile.</string>
       </property>
       <property name="text">
        <string>Add Existing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="whatsThis">
        <string>I've changed my mind, close this form.</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
